handlers/user.go

```diff
@@ -328,10 +328,10 @@ func getValidatorTableData(userId uint64) (interface{}, error) {
 	}{}
 
 	err := db.FrontendDB.Select(&validatordb, `
-SELECT ENCODE(uvt.validator_publickey::bytea, 'hex') AS pubkey, us.event_name, extract( epoch from last_sent_ts)::Int as last_sent_ts, us.event_threshold
-FROM users_validators_tags uvt
-LEFT JOIN users_subscriptions us ON us.event_filter = ENCODE(uvt.validator_publickey::bytea, 'hex') AND us.user_id = uvt.user_id
-WHERE uvt.user_id = $1;`, userId)
+	SELECT ENCODE(uvt.validator_publickey::bytea, 'hex') AS pubkey, us.event_name, extract( epoch from last_sent_ts)::Int as last_sent_ts, us.event_threshold
+		FROM users_validators_tags uvt
+		LEFT JOIN users_subscriptions us ON us.event_filter = uvt.validator_publickey::bytea AND us.user_id = uvt.user_id
```

Comment:
event_filter is a string and you are comparing it to a bytea `us.event_filter = uvt.validator_publickey::bytea`. I don't think this change is necessary.

Reply:
Did no changes in this row